resources/chisel/textures/blocks/glowstone/slanted_glow.png.mcmeta

```diff
@@ -0,0 +1,11 @@
+{
+    "ctm": {
+        "ctm_version": 1,
+        "layer": "BLOOM",
+        "type": "pattern",
+        "extra": {
+            "light": 16,
```

Comment:
PR is still full of incorrect light level `16`. Fix it please.

You can use VSCode "search" feature to easily find all the wrong numbers
![](https://i.imgur.com/uBJQEva.png)